sections/conclusion/future-work.tex

```diff
@@ -0,0 +1,40 @@
+\section{Future work}
+This section will discuss the aspects of the project that were not implemented, but were deemed to improve the system as a whole if implemented.
+
+\subsubsection{Dead Reckoning}
+Dead Reckoning is a technique described in \autoref{sec:dead-reckoning} used to smooth out player movement over networking through predicting their next position.
+This could be a possible improvement on the real-time feeling of the game, leading to an improved player experience.
+As such, it is a prime candidate for implementation in the future, in order to improve the game.
+
+\subsubsection{Zeroconf}
+Zeroconf was briefly discussed as a zero configuration management protocol in \autoref{sec:accessonnetwork} as an alternative to having players enter the IP of the host they wanted to connect to.
+Manually typing the IP of the host is not a viable solution for a deployed game in use in diverse locations.
+As such, introducing a lobby system through the use of Zeroconf would enhance the usability of the game for players.
+During the course of the project, some developers attempted to implement a lobby system using this technology.
+A working implementation on an android application was made, however, certain incompatibility issues were encountered when attempting to integrate this with Unity.
+After attempting to correct these without luck, it was decided that the time spent on this aspect was not worth it for the creation of the MVP.
+Ultimately, this functionality is needed for future versions of the game, and should be pursued.
+
+\subsubsection{Replay functionality}
+While discussing the scope of the project, a replay functionality was considered but quickly deferred until later as a non-essential aspect of the game.
+This replay functionality would be able to showcase highlights of a game when a game had completed, such as showing goals or certain interesting moments.
+This would be an interesting feature for the players of the game, and could help make playing the game more fun.
+As such, it would be an interesting feature to add in a possible future work scenario.
+
+\subsubsection{More aspects to the game}
+Several other aspects to make the game more interesting were also discussed in the same vein as the replay functionality.
```

Comment:
Man kunne godt nævne muligheden for en virtuel bold i sammenhæng med dette

Reply:
true